demo/addons/godot-openxr/scenes/first_person_controller_vr.gd

```diff
@@ -3,12 +3,22 @@ extends ARVROrigin
 signal initialised
 signal failed_initialisation
 
+# Also add signals so we can have other parts of the application react to this.
+signal session_begun
```

Comment:
@m4gr3d I'm chaining these signals so we can easily hook them up in the UI so game code can react on them.